### PR TITLE
NN-2827 tidy up

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/health/QueueHealth.kt [hunk 1]
```diff
@@ -1,13 +1,9 @@
 package uk.gov.justice.digital.hmpps.keyworker.services.health
 
 import com.amazonaws.services.sqs.AmazonSQS
-import com.amazonaws.services.sqs.model.GetQueueAttributesRequest
-import com.amazonaws.services.sqs.model.GetQueueAttributesResult
-import com.amazonaws.services.sqs.model.GetQueueUrlResult
-import com.amazonaws.services.sqs.model.QueueAttributeName
+import com.amazonaws.services.sqs.model.*
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/health/QueueHealth.kt:4]: prefer wildcard imports

Comment (Nimphal) [inline src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/health/QueueHealth.kt:4]: I believe we're avoiding wildcard imports now (I know this is the IDE doing things).

Comment (JamieDraperUK) [reply src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/health/QueueHealth.kt:4]: Dam it!

File: src/test/java/uk/gov/justice/digital/hmpps/keyworker/controllers/KeyworkerServiceControllerTest.kt [hunk 2]
```diff
@@ -20,7 +20,7 @@ import uk.gov.justice.digital.hmpps.keyworker.services.KeyworkerAutoAllocationSe
 import uk.gov.justice.digital.hmpps.keyworker.services.KeyworkerMigrationService
 import uk.gov.justice.digital.hmpps.keyworker.services.KeyworkerService
 import uk.gov.justice.digital.hmpps.keyworker.services.PrisonSupportedService
-import java.util.Optional
+import java.util.*
```
Comment (Nimphal) [inline src/test/java/uk/gov/justice/digital/hmpps/keyworker/controllers/KeyworkerServiceControllerTest.kt:23]: This is a weird one!

**Pull request merged**